.github/actions/start-opensearch-with-one-plugin/action.yml

```diff
@@ -0,0 +1,126 @@
+name: 'Start OpenSearch with One Plugin'
+description: 'Downloads latest build of OpenSearch, installs a plugin, executes and script and then starts OpenSearch on localhost:9200'
+
+inputs:
+  opensearch-version:
+    description: 'The version of OpenSearch that should be used, e.g "3.0.0"'
+    required: true
+
+  plugin-name:
+    description: 'The the name of the plugin to use, such as security-dashboards-plugin'
```

Comment:
nit: The appears twice in the beginning here

`security-dashboards-plugin` is a plugin for OpenSearch Dashboards, but this action looks like it starts an `opensearch` node. Should this reference an `opensearch` plugin?